client/src/Common/Layout.js

```diff
@@ -56,6 +57,11 @@ class CustomLayout extends React.Component{
                 theme="light"
                 mode="horizontal"
                 style={{lineHeight: '64px'}} >
+<Menu.Item
+    key="booksList"
+    style={{float: 'right'}}>
```

Comment:
I think it should be placed in center possision like rest of tabs so set style={{float: 'center', marginLeft: '2%'}}>